frontend/containers/App/App.jsx

```diff
@@ -90,7 +91,7 @@ class App extends Component {
     // -- eb, 01/02/2017
 
     return (
-      <div>
+      <div style="height: 100%;">
```

Comment:
They can be. App doesn't currently have a style sheet and this is the only styling it'll need, but I'm happy to move it to CSS.